core/substrait-core.csproj

```diff
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>

```

Comment:
The target framework for a library should be the minimum possible version you can target.  It forces any user of this library to be using this version or newer.  Since this is just wrapping protobuf this should match whatever your dependencies require.  That appears to be both `.NETStandard 1.1` and `.NETFramework 4.5`.  So I'm pretty sure you should be able to get away with just `.NETStandard 1.1`.

Reply:
Thanks @westonpace . I have changed it accordingly and compilation went through indeed.